src/client/pages/OfferNew/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react'
+import { colorsV3 } from '@hedviginsurance/brand'
+import { AnimateSharedLayout } from 'framer-motion'
+import styled from '@emotion/styled'
+import { MEDIUM_SCREEN_MEDIA_QUERY } from 'utils/mediaQueries'
+import { Tab } from './Tab'
+
+const TabGroup = styled.div`
+  border-bottom: 1px solid ${colorsV3.gray500};
```

Comment:
```suggestion
  border-bottom: 2px solid ${colorsV3.gray500};
```

...however, from Figma it seems like this is implemented as a box-shadow, have you tried how that looks?

```
box-shadow: 0px 2px 0px #EAEAEA;
```